### PR TITLE
fix(api-client): Team & Payment type update

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: packages/api-client/src/main/team/MemberRole.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+enum MemberRole {
+  ADMIN = 'admin',
+  PARTNER = 'partner',
+  MEMBER = 'member',
+  OWNER = 'owner',
+}
```
Comment (ffflorian) [inline packages/api-client/src/main/team/MemberRole.ts]: sorting

File: packages/api-client/src/main/team/payment/PaymentBillingData.ts [hunk 2]
```diff
@@ -16,12 +16,14 @@
...
  *
  */
+import {SupportedCurrency} from './SupportedCurrency';
 
 interface PaymentBillingData {
   city: string;
   company: string;
   country: string;
   firstname: string;
+  currency?: SupportedCurrency;
```
Comment (ffflorian) [inline packages/api-client/src/main/team/payment/PaymentBillingData.ts]: sorting

File: packages/api-client/src/main/team/payment/PaymentData.ts [hunk 3]
```diff
@@ -18,24 +18,42 @@
...
 
 import {InvoiceData, PaymentPlan} from './';
+import {PaymentPlanID} from './PaymentPlan';
+
+enum PaymentStatus {
+  TRAILING = 'trialing',
+  ACTIVE = 'active',
+  PAST_DUE = 'past_due',
+  CANCELED = 'canceled',
+  UNPAID = 'unpaid',
```
Comment (ffflorian) [inline packages/api-client/src/main/team/payment/PaymentData.ts]: sorting

File: packages/api-client/src/main/team/payment/PaymentData.ts [hunk 4]
```diff
@@ -18,24 +18,42 @@
...
+  expYear: number;
+  holder: string;
+  zip: string;
+}
+
+interface PaymentSuspension {
+  invoice: string;
+  graceEnding: number;
+  created: number;
+}
```
Comment (ffflorian) [inline packages/api-client/src/main/team/payment/PaymentData.ts]: sorting

File: packages/api-client/src/main/team/payment/PaymentStripeCharge.ts [hunk 5]
```diff
@@ -21,6 +21,12 @@
 
 import {SupportedCurrency} from '../payment/';
 
+enum PaymentStripeChargeStatus {
+  SUCCEEDED = 'succeeded',
+  PENDING = 'pending',
+  FAILED = 'failed',
+}
```
Comment (ffflorian) [inline packages/api-client/src/main/team/payment/PaymentStripeCharge.ts]: sorting

**Comment by ffflorian** [comment]
This is more of a feature, no?

File: packages/api-client/src/main/team/payment/PaymentStripeInvoice.ts [hunk 6]
```diff
@@ -24,6 +26,7 @@ interface PaymentStripeInvoice {
   date: number;
   paid: boolean;
   closed: boolean;
+  currency: SupportedCurrency;
```
Comment (bennyn) [inline packages/api-client/src/main/team/payment/PaymentStripeInvoice.ts]: sorting ;)

**Pull request merged**